.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,26 +13,26 @@ about: Create a report to help us improve
 
 ---
 
-## :writing_hand: Describe the bug
+## 📖 Description
 <!-- A clear and concise description of what the bug is. -->
 
-## :bomb: Steps to reproduce
+## ℹ️ Environment
+- **Device**: <!-- e.g. Pixel 5 -->
+- **OS**: <!-- e.g. 9.0.0 -->
+- **Library version**: <!-- e.g. Sentinel 1.3.0). -->
```

Comment:
Since I assume a lot of people won't update the bug report after creating their library/repo, it's best to keep it library-agnostic IMO

```suggestion
- **Library version**: <!-- e.g. 1.3.0). -->
```